pom.xml

```diff
@@ -232,7 +250,7 @@
           <plugin>
             <groupId>org.sonatype.plugins</groupId>
             <artifactId>nexus-staging-maven-plugin</artifactId>
-            <version>${nexusplugin.version}</version>
+            <version>${nexusstagingmaven.version}</version>
```

Comment:
Couldn't find the definition of `nexusstagingmaven.version`. I would remove the profiles `deploy-central` and `sign`, since they are already defined here: https://gitlab.gistools.geog.uni-heidelberg.de/giscience/big-data/ohsome/parent/blob/2.2/pom.xml#L118-168

Reply:
Found the definition. If you want you can still remove the profiles, but this is resolved.